library/src/validations/ulid/ulid.ts

```diff
@@ -2,15 +2,15 @@ import type { ErrorMessage, PipeResult } from '../../types.ts';
 import { getOutput, getPipeIssues } from '../../utils/index.ts';
 
 /**
- * Creates a validation functions that validates a [ULID](https://github.com/ulid/spec).
+ * Creates a validation function that validates a [ULID](https://github.com/ulid/spec).
  *
  * @param error The error message.
  *
  * @returns A validation function.
  */
 export function ulid<TInput extends string>(error?: ErrorMessage) {
   return (input: TInput): PipeResult<TInput> =>
-    !/^[0-9A-HJKMNPQ-TV-Z]{26}$/i.test(input)
+    !/^[\da-hjkmnp-tv-z]{26}$/iu.test(input)
```

Comment:
I think that the `q` in the regex has been lost here or?

Reply:
`p-t` includes q `p -> qrs <- t`

Reply:
My bad 😄